UNI_EXT/SoulstormHandlers.cpp

```diff
@@ -0,0 +1,17 @@
+//#include "SoulstormPointers.h"
+#include "stdafx.h"
+
+DWORD WINAPI MainThread(LPVOID param)
+{
+	bool inGame = false;
+	while (true)
+	{
+		if (GetAsyncKeyState(VK_F6) & 0x80000)
+		{
+			MessageBoxA(NULL, "F6 Pressed", "F6 Pressed", MB_OK);
```

Comment:
Is this still required? It looks like some old debug code that reacts on your F6 keypress.